packages/sit-onyx/src/components/OnyxAccordionItem/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { DensityProp } from "../../composables/density";
+import type { SkeletonInjected } from "../../composables/useSkeletonState";
+
+export type OnyxAccordionItemProps = DensityProp & {
```

Comment:
Would be nice if the open state can be controlled as prop so you could e.g. open it programatically or set it open initially on page load.

You don't have to do it in this PR